classes/class-wc-connect-service-settings-store.php

```diff
@@ -110,6 +110,40 @@ public function can_user_manage_payment_methods() {
 				( is_a( $master_user, 'WP_User' ) && $current_user->ID === $master_user->ID );
 		}
 
+		/**
+		 * Gets the store's location settings.
+		 *
+		 * @return array
+		 */
+		public function get_endshipper_address() {
+			$email       = '';
+			$master_user = WC_Connect_Jetpack::get_master_user();
+
+			if ( is_a( $master_user, 'WP_User' ) ) {
+
+				$connected_data = WC_Connect_Jetpack::get_connected_user_data( $master_user->ID );
+
+				if ( is_array( $connected_data ) && ! empty( $connected_data['email'] ) ) {
+					$email = $connected_data['email'];
+				}
+			}
```

Comment:
Let's extract this part to a separate method, e.g. `get_end_shipper_email()`.

Reply:
Sure, let me do that. I think it will make the code cleaner.